### DISCUSSION TITLE
How to get `save_dir` of CheckpointSaver from MLFlowHandler's artifact path?

### DISCUSSION THREAD

**Discussion by whikwon** [Project-MONAI/MONAI#7514] [Q&A]
I want to use CheckpointSaver and MLFlowHandler together, but the MLFlowHandler starts when the trainer starts, so the CheckpointSaver cannot determine the save_dir. Is there a way to solve this problem?

Thank you in advance.

**Comment by KumoLiu** [comment]
Hi @binliunls, could you please share some comments here? Thanks.

**Comment by binliunls** [comment]
Hi @whikwon , could you please explain more details about your question? I am not clear about what you want here and what the use case is. Do you want the `CheckpointSave` to determine the output dir for MLFlow tracking files?

Thanks
Bin

**Comment by whikwon** [comment]
Hi @binliunls! 
I am using MlflowHandler, and I want to save Checkpoints. Storing them as Mlflow artifacts seems to be the best option, but I couldn't find the function. 
I want to inquire if there is a way to use them with CheckpointSaver or any other possible methods.
